lib/Elastica/Aggregation/AbstractAggregation.php

```diff
@@ -18,6 +18,11 @@ abstract class AbstractAggregation extends Param implements NameableInterface
      */
     protected $_aggs = [];
 
+    /**
+     * @var array|null Metadata belonging to this aggregation
+     */
+    protected $_meta;
```

Comment:
Do we need a separate `_meta` key here or could we just use the internal `setParam`?

Reply:
After going over the code in `AbstractAggregation::toArray()` I agree the problem could be solved by using `AbstractAggregation::_setRawParam($key, $value)`.